tools/batch_run_pipeline_qc_face_script.pl

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+batch_run_pipeline_qc_face_script.pl -- Run C<pipeline_qc_deface.pl> in batch mode
+
+=head1 SYNOPSIS
+
+perl batch_run_pipeline_qc_face_script.pl [-profile file] [-out_basedir directory] < list_of_files.txt
+
+Available options are:
+
+-profile: name of config file in ../dicom-archive/.loris_mri (typically called prod)
+
+-out_basedir: path to the output base directory where the jpg will be created
+
+=head1 DESCRIPTION
+
+This script runs the creation of 3D rendering QC images on multiple MINC files.
```

Comment:
This script creates the 3D rendering of QC images on...

Reply:
done